.github/workflows/ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
             cargo update
         fi
         cargo test --release
-        cargo test --release --features=no-threads
+        env BLST_TEST_NO_STD= cargo test --release
```

Comment:
This is not how Rust ecosystem does it. It should be controlled by features (there is `--no-default-features` for instance). See #150 on how to do it in idiomatic way.

Reply:
> This is not how Rust ecosystem does it.

I don't really care. I want to exercise the no_std build on an std platform **without** offering an idiomatic way.

Reply:
What is the point? It just confuses the heck out of people.

I get that in other languages that might be a norm, but in Rust there are conventions that make like so much easier, I don't quite understand the rationale behind deliberately making it harder instead :confused: 

IMHO Rust API should look like idiomatic Rust, Golang API should look like idiomatic Golang and so on for every language to the degree possible.

Reply:
> What is the point?

??? It says just one line above, "exercise the no_std build on an std platform." It's not for users, but for us as a way to ensure that no_std doesn't go under the CI radar unchecked.